src/docs/admin/ProActiveAdminGuide.adoc

```diff
@@ -570,28 +570,103 @@ pa.scheduler.core.authentication.loginMethod=SchedulerLDAPLoginMethod
 By default, the ProActive Resource Manager stores users accounts, passwords, and group memberships (user or admin), in two files:
 
 * users and passwords accounts are stored in `PROACTIVE_HOME/config/authentication/login.cfg`.
-Each line has to follow the format *user:passwd*. The default `login.cfg` file is given hereafter:
+Each line has to follow the format *user:encypted password*.
+The default accounts in `login.cfg` file corresponds to the following passwords, which are encrypted in the actual login.cfg file:
 +
-[source]
+.login.cfg
```

Comment:
Better naming (e.g. unencrypted_login.cfg) to prevent confusion with existing _login.cfg_?